inception-search-mtas/src/test/resources/log4j2.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="WARN">
-
   <Appenders>
     <Console name="ConsoleAppender" target="SYSTEM_OUT">
       <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %level{length=5} %logger{1} - %msg%n" />
     </Console>
   </Appenders>
 
   <Loggers>
-    <Logger name="mtas" level="WARN"/>
-    <Root level="warn">
+    <Logger name="mtas" level="TRACE"/>
+    <Logger name="de.tudarmstadt.ukp.inception.search.index.MtasUimaParser" level="TRACE"/>
```

Comment:
I would prefer not to activate trace logging in the release, we moved some of the debug calls to trace because it really spammed the logs.

Reply:
All of this is work in progress, the test is still not running. Actually this log4j2 was commited by mistake, it was supposed to run in trace only in my local run.